core-integration-test/src/test/groovy/org/openstack4j/api/AbstractSpec.groovy

```diff
@@ -18,7 +18,7 @@ import co.freeside.betamax.tape.yaml.TapePropertyUtils
 abstract class AbstractSpec extends Specification {
 
     def static String MODULEROOT = Paths.get("").toAbsolutePath().getParent().toString()
-    def static String TAPEROOT = "/" + MODULEROOT + "src/test/resources/tapes/"
+    def static String TAPEROOT = MODULEROOT + "/" + "src/test/resources/tapes/"
```

Comment:
@olivergondza I think it should be TAPEROOT = "/" + MODULEROOT + "/src/test/resources/tapes/"

With JDK 1.7 that reduces the amount of errors in the integration-test module to 3 NPE's caused by UnknownHostExceptions. Very strange. I'm currently looking into it.